kasa/smart/modules/firmware.py

```diff
@@ -22,36 +23,39 @@
 _LOGGER = logging.getLogger(__name__)
 
 
-class DownloadState(BaseModel):
+@dataclass
+class DownloadState(DataClassDictMixin):
     """Download state."""
 
     # Example:
     #   {'status': 0, 'download_progress': 0, 'reboot_time': 5,
     #    'upgrade_time': 5, 'auto_upgrade': False}
     status: int
-    progress: int = Field(alias="download_progress")
+    progress: int = field(metadata=field_options(alias="download_progress"))
     reboot_time: int
     upgrade_time: int
     auto_upgrade: bool
 
 
-class UpdateInfo(BaseModel):
+@dataclass
+class UpdateInfo(DataClassDictMixin):
     """Update info status object."""
 
-    status: int = Field(alias="type")
-    version: str | None = Field(alias="fw_ver", default=None)
+    status: int = field(metadata=field_options(alias="type"))
```

Comment:
```suggestion
    status: Annotated[int, Alias("type")]
```
What do you think about using `Annotated` https://github.com/Fatal1ty/mashumaro?tab=readme-ov-file#field-aliases for aliases?

Reply:
I like it, it's a bit more concise

Reply:
Done